packages/babel-generator/src/node/parentheses.js

```diff
@@ -80,55 +75,35 @@ export function Binary(node: Object, parent: Object): boolean {
 }
 
 export function BinaryExpression(node: Object, parent: Object): boolean {
-  if (node.operator === "in") {
+  if (
```

Comment:
You could simply return the conditional you are checking rather since this function essentially reads as `if (true) { return true; } else { return false; }`.

So: `return node.operator === "in" && ...`